vips/vip-515/bsctestnet.ts

```diff
@@ -0,0 +1,346 @@
+import { BigNumber, BigNumberish } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { ProposalMeta, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { bsctestnet } = NETWORK_ADDRESSES;
+
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+export const Actions = {
+  MINT: 0,
+  BORROW: 2,
+  ENTER_MARKET: 7,
+};
+
+export const PROTOCOL_SHARE_RESERVE = "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3";
+export const REDUCE_RESERVES_BLOCK_DELTA_BSC = "28800";
+export const CONVERSION_INCENTIVE = 1e14;
+
+export const asBNBMarketSpec = {
+  vToken: {
+    address: "0x73F506Aefd5e169D48Ea21A373B9B0a200E37585",
+    name: "Venus asBNB",
+    symbol: "vasBNB",
+    underlying: {
+      address: "0xc625f060ad25f4A6c2d9eBF30C133dB61B7AF072",
+      symbol: "asBNB",
+      decimals: 18,
+    },
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: bsctestnet.UNITROLLER,
+    isLegacyPool: true,
+  },
+  interestRateModel: {
+    model: "jump",
+    baseRatePerYear: "0",
+    multiplierPerYear: "0.09",
+    jumpMultiplierPerYear: "2",
+    kink: "0.5",
+  },
+  initialSupply: {
+    amount: parseUnits("0.14", 18), // 0.14 asBNB
+    vTokensToBurn: parseUnits("0.14", 8), // 0.14 vasBNB
+    vTokenReceiver: bsctestnet.VTREASURY,
+  },
+  riskParameters: {
+    supplyCap: parseUnits("2000", 18), // 2000 asBNB
+    borrowCap: parseUnits("0", 18), // 0 asBNB
+    collateralFactor: parseUnits("0.72", 18),
+    reserveFactor: parseUnits("0.1", 18),
+  },
+};
+
+export const MockedUSDF = "0xC7a2b79432Fd3e3d5bd2d96A456c734AB93A0484";
+export const MockedUSDFOracleAddress = "0xEca2605f0BCF2BA5966372C99837b1F182d3D620"; // Chainlink Price Feed USDT/USD on BSC Testnet
+
+export const USDFMarketSpec = {
+  vToken: {
+    address: "0x140d5Da2cE9fb9A8725cabdDB2Fe8ea831342C78",
+    name: "Venus USDF",
+    symbol: "vUSDF",
+    underlying: {
+      address: MockedUSDF,
+      symbol: "USDF",
+      decimals: 18,
+    },
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: bsctestnet.UNITROLLER,
+    isLegacyPool: true,
+  },
+  interestRateModel: {
+    model: "jump",
+    baseRatePerYear: "0",
+    multiplierPerYear: "0.09",
+    jumpMultiplierPerYear: "2",
+    kink: "0.5",
+  },
+  initialSupply: {
+    amount: parseUnits("20100", 18), // 20100 USDF
+    vTokensToBurn: parseUnits("100", 8), // 100 vUSDF
+    vTokenReceiver: "0xa8c0C6Ee62F5AD95730fe23cCF37d1c1FFAA1c3f",
```

Comment:
On testnet, I think it's better if we transfer to the VTreasury the vTokens that we are not burning